api/v1alpha1/mariadb_types.go

```diff
@@ -70,6 +70,7 @@ func (p *PodDisruptionBudget) Validate() error {
 type ServiceTemplate struct {
 	Type        corev1.ServiceType `json:"type,omitempty"`
 	Annotations map[string]string  `json:"annotations,omitempty"`
+	ExternalTrafficPolicy [string]string `json:"externaltrafficpolicy,omitempty"`
```

Comment:
This map notation doesn't seem to be accepted in our GolangCI Lint rules, could you change it to `map[string]string`?

Also, the fields should be in camel case, could you update this also?: `externalTrafficPolicy`